### ISSUE TITLE
ikos fails on the Eigen3 example: unexpected llvm::BinaryOperator (opcode: fmul)

### ISSUE THREAD

**Issue by yurivict** [NASA-SW-VnV/ikos#87]
Example 2 from https://eigen.tuxfamily.org/dox/GettingStarted.html fails:
```
$ ikos -o eig eig.cpp -I /usr/local/include/eigen3/
[*] Compiling eig.cpp
[*] Running ikos preprocessor
[*] Running ikos analyzer
[*] Translating LLVM bitcode to AR
ikos-analyzer: /tmp/ikos-Ebky1g/eig.pp.bc: error: unexpected llvm::BinaryOperator (opcode: fmul)
ikos: error: a run-time error occurred
```

**Comment by arthaud** [comment]
This is a duplicate of #55

**Comment by arthaud** [comment]
I just fixed #55, please tell me if the issue is fixed.

**Comment by yurivict** [comment]
It still fails, now with a different error message:
```
ikos-analyzer: /tmp/ikos-qCTq_n/ex.pp.bc: error: unexpected llvm::BitCastInst
```

**Comment by arthaud** [comment]
This is tracked by #56. I will look for a fix soon.

**Comment by arthaud** [comment]
This should be fixed by ab441860ea2a99582324b71896163327d7a77e00.
